user_docs/en/userGuide.md

```diff
@@ -2388,6 +2390,39 @@ By default only three modes are included.
 Note that it is necessary to check at least one mode.
 This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.
 
+##### Applications volume adjuster status {#AppsVolumeAdjusterStatus}
+
+This combo box allows you to select the status of the applications volume adjuster.
+The applications volume adjuster allows you to adjust volume of all other applications except for NVDA or mute them with a single keystroke.
+
+Possible values are:
+
+* Disabled: NVDA doesn't interfere with volume levels of other applications.
+* Enabled: volume of other applications can be adjusted via [other applications volume slider](#OtherAppVolume).
+* Muted: all other applications are muted.
+
+The following keyboard command can also be used from anywhere:
+
+| Name | Key | Description |
+|---|---|---|
+| Toggle Applications volume adjuster status | `NVDA+alt+delete` | Cycles through options of Applications volume adjuster status. |
+If this option is set to Muted, then its value will not be stored and it will be restored to Enabled upon next restart of NVDA.
```

Comment:
```suggestion
| Toggle "Applications volume adjuster" status | `NVDA+alt+delete` | Cycles through statuses for "Applications volume adjuster" |

If this option is set to muted, then its value will not be stored and it will be restored to enabled upon next restart of NVDA.
```